sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/validator/PlatformKey.java

```diff
@@ -22,7 +23,7 @@
  *
  * <p>The validator accepts <code>null</code> values.
  */
-@Target({FIELD, PARAMETER})
+@Target({ METHOD, FIELD, ANNOTATION_TYPE, CONSTRUCTOR, PARAMETER, TYPE_USE })
```

Comment:
Is there anything that can be validated at the constructor or are the Hibernate docs missing something?

```suggestion
@Target({ FIELD, METHOD, PARAMETER, ANNOTATION_TYPE, TYPE_USE })
```
Taken from here: https://docs.jboss.org/hibernate/stable/validator/reference/en-US/html_single/?v=7.0#validator-customconstraints-constraintannotation

Note: Spotless strikes back again.

Reply:
I took the list from org.hibernate.validator.internal.constraintvalidators.bv.number.sign.NegativeOrZeroValidatorForBigInteger but I am fine with the suggestion